.github/workflows/schema.yml

```diff
@@ -0,0 +1,44 @@
+name: Update schema
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * 1-5" # Weekdays
+
+jobs:
+  update:
+    if: github.repository == "aws/serverless-application-model"
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+      pull-requests: write
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+
+      - name: Update schema
+        id: schema
+        run: |
+          make init
+          make schema-all
+          git diff --exit-code || echo "changed=true" >> $GITHUB_OUTPUT
```

Comment:
what does this line do? if only there is more comment here

Reply:
Checks whether anything changed. I thought the `diff` and `changed` would be clear enough

Reply:
I think I am getting it after reading the `if` below

Reply:
```suggestion
          # Set gh action condition "steps.schema.outputs.changed" to true if there is diff:
          git diff --exit-code || echo "changed=true" >> $GITHUB_OUTPUT
```